app/src/main/java/com/smelnykov/homework_1/AuthActivity/AuthActivity.kt

```diff
@@ -56,18 +66,34 @@ class AuthActivity : AppCompatActivity() {
             }
 
             if (isAllFieldsCorrect) {
-                sendEmail(editTextEmail.text.toString())
+                registerAndChangeActivity(
+                    editTextEmail.text.toString(),
+                    editTextPassword.text.toString()
+                )
             }
         }
     }
 
     /**
-     * Method for sending email to My Profile Activity.
-     * @param email email which will be send
+     * Method for saving email, password and change activity to My Profile Activity.
+     * @param email email which will be saved
+     * @param password password which will be saved
+     * */
+    private fun registerAndChangeActivity(email: String, password: String) {
+        val editor = sharedPreference.edit()
+        editor.putString(Constants.EMAIL_AUTH_KEY, email)
+        editor.putString(Constants.PASSWORD_AUTH_KEY, password)
+        editor.apply()
+
+        val intent = Intent(this@AuthActivity, MyProfileActivity::class.java)
+        startActivity(intent)
+    }
+
+    /**
+     * Method for autologin redirection to my profile activity.
      * */
-    private fun sendEmail(email: String) {
+    private fun authLogin() {
```

Comment:
auth == login 😅

Reply:
Yeah, I made mistake😅 I meant auto, not auth